packages/adapter-d1/src/conversion.ts

```diff
@@ -8,6 +8,8 @@ import {
 
 export type Value = null | string | number | object
 
+export const absoluteBigint = (n) => (n < 0n ? -n : n)
```

Comment:
Not used at the moment, I thought it was good to keep in case we decide to do something with it